src/integrations/terminal/TerminalProcess.ts

```diff
@@ -260,61 +199,18 @@ export class TerminalProcess extends EventEmitter<TerminalProcessEvents> {
 			}
 			this.isHot = false
 
-			console.log(`[TerminalProcess] Stream ended for command: "${command}"`)
-			console.log(`[TerminalProcess] Final output length: ${this.fullOutput.length} characters`)
-
-			// Check if this looks like a command that completed vs one that's still running
-			const quickCommands = ["cd ", "pwd", "ls ", "echo ", "mkdir ", "touch ", "rm ", "cp ", "mv "]
-			const isQuickCommand = quickCommands.some((cmd) => command.startsWith(cmd) || command.includes(" && " + cmd))
-
-			// Check if output suggests a long-running process
-			const longRunningIndicators = [
-				"listening on",
-				"server running",
-				"started on",
-				"watching for",
-				"compiled successfully",
-				"webpack",
-				"vite",
-				"nodemon",
-				"dev server",
-				"press ctrl",
-				"to quit",
-				"to exit",
-				"to stop",
-			]
-			const hasLongRunningOutput = longRunningIndicators.some((indicator) =>
-				this.fullOutput.toLowerCase().includes(indicator),
-			)
+			this.emit("completed")
+			this.emit("continue")
+		} else {
+			// no shell integration detected, we'll fallback to running the command and capturing the terminal's output after some time
+			terminal.sendText(command, true)
 
-			// Check if this is likely a command that starts a server or long-running process
-			const longRunningCommands = ["npm run", "npm start", "yarn", "node ", "python ", "serve", "dev", "watch"]
-			const isLongRunningCommand = longRunningCommands.some((cmd) => command.includes(cmd))
+			// wait 3 seconds for the command to run
+			await new Promise((resolve) => setTimeout(resolve, 3000))
```

Comment:
Arbitrary wait times are not a good practice.  We should instead find a way to await something that isn't arbitrary.